code/game/objects/items/robot/robot_items.dm

```diff
@@ -823,14 +823,20 @@
 	stored.forceMove(get_turf(usr))
 	return
 
+/**
+* Attack_self will pass for the stored item. 
+*/
 /obj/item/borg/apparatus/attack_self(mob/living/silicon/robot/user)
 	if(!stored)
 		return ..()
-	if(user.client?.keys_held["Alt"])
-		stored.forceMove(get_turf(user))
-		return
 	stored.attack_self(user)
 
+//Alt click drops the stored item.
+/obj/item/borg/apparatus/AltClick(mob/living/silicon/robot/user)
+	if(!stored)
+		return ..()
+	stored.forceMove(get_turf(user))
```

Comment:
Also this isn't observer/deadborg safe.

Reply:
As in it being triggered when someone is observing? Would an if check suffice?

Reply:
If someone observes a borg and alt clicks the ui yes.

Reply:
i put a simple !issilicon check. Let me know if its sufficient.